### PR TITLE
Des 61 dev make responsive dashboard

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

File: src/components/organims/home/HomeViewUser.jsx/HomeViewUser.jsx [hunk 1]
```diff
@@ -16,7 +14,7 @@ export const HomeViewUser = () => {
           maiores molestias unde nostrum earum?
```
Comment (DanielCalleCO) [inline src/components/organims/home/HomeViewUser.jsx/HomeViewUser.jsx:14]: Not to fix... But, what is this? hehehe

File: public/index.html [hunk 2]
```diff
@@ -15,6 +15,9 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
```
Comment (DanielCalleCO) [inline public/index.html:16]: Do you finally decide to let these comments on the index.html?

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]
Did you delete the folder "origen" and its content?

File: src/components/molecules/homeContent/dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import './dashboard.css';
+
+export const Dashborad = () => {
+  return (
+    <>
+      <div className="content-welcome">
+        <div className="container-welcome">
+          <div className="container-text-welcome">
+            <h2 className="welcome-title">Bienvenido/a, Daniel</h2>
```
Comment (DanielCalleCO) [inline src/components/molecules/homeContent/dashboard/Dashboard.jsx:9]: I suppose that this "Daniel" will be changed by the name (variant) of each student, right?

**Review by DanielCalleCO — APPROVED** [review]

**Review by SthefaniaEcheverry — APPROVED** [review]

**Pull request merged**